src/components/FileUploaderDetails/UploaderProgressBar.part.tsx

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
+
+import { Icon, IconName } from '../Icon';
+import { ProgressStatus, TranslationLabels } from './FileUploaderDetails.types.part';
+import { defaultLabels, iconNames } from './helpers';
 import styled, { reStyled } from '../../utils/styled';
-import { distance } from '../../distance';
+
```

Comment:
I am not sure what's up with these empty lines in here (and other files). Can you please remove them? Why change the import blocks?

Reply:
I have plugin in vs code which alphabetically sorts imports.

Reply:
Yep, but can you either turn it off or configure it to not insert blank lines? We always have a single block of imports. Also we have a sorting that usually does in order

- all (asterisk) imports
- default imports
- named imports

where external packages `'foo'` are ranked higher than internal ones (`'./foo'`). I have no problem with changing the order slightly, but inserting these blank lines is rather disturbing because I (personally) think some meaningful LOC starts already.